schema/url/visualizations.raml

```diff
@@ -36,6 +36,24 @@ types:
       xMax:
         type: string
 
+  LatitudeViewport:
```

Comment:
Suggestion - min and max don't seem specific to Latitutde. Maybe this could be a more general Viewport?

Reply:
May be reasonable to have xAxisViewport be a more general viewport if we go that direction, too?

Reply:
i made the xaxis one the way i did in case we end up w stats charts w y axis viewports as well... but you may be right. we could have NumericViewport w min and max